Src/EB/AMReX_EB2_2D_C.cpp

```diff
@@ -310,6 +311,9 @@ int build_faces (Box const& bx, Array4<EBCellFlag> const& cell,
                 if (fy(i  ,j+1,0) == Type::irregular) { ++ncuts; }
                 if (ncuts > 2) {
                     Gpu::Atomic::Add(dp,1);
+                    if (plt_multiple_cuts){
```

Comment:
bring this outside of the `if (ncuts > 2)` scope. That way ncuts = 2 will show regular cut cells, ncuts > 2 multicuts, and ncuts = 0 cells that are either completely covered or completely fluid.